src/main/java/blackjack/domain/Players.java

```diff
@@ -0,0 +1,84 @@
+package blackjack.domain;
+
+import blackjack.dto.CardInfo;
+import blackjack.dto.NameInfo;
+import blackjack.dto.ScoreInfo;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Players {
+
+    private final List<Player> players;
+
+    private int activePlayerIndex;
+
+    public Players(final List<Player> players) {
+        this.players = players;
+        this.activePlayerIndex = 0;
+    }
+
+    public static Players create(final List<String> names) {
+        return new Players(
+            names.stream()
+                .map(name -> new Player(name, new Cards(new ArrayList<>())))
+                .collect(Collectors.toList())
+        );
+    }
+
+    public void initializeDeck(final Deck deck) {
+        players.stream().forEach(player -> player.initializeDeck(deck));
```

Comment:
stream을 쓰지않고, `Collection.forEach()`를 활용할 수 있습니다!

```suggestion
        players.forEach(player -> player.initializeDeck(deck));
```

Reply:
감사합니다!! 굳이 `stream`객체를 생성하지 않고 바로 반복을 돌리는 것이 더 좋겠네요!!